csharp/.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I'm not sure we want to push this. Another option would be to `.gitignore` this file.

Reply:
I don't think we should push this - add to `.gitignore`